pandas-stubs/core/indexes/interval.pyi

```diff
@@ -310,59 +310,17 @@ class IntervalIndex(ExtensionIndex[IntervalT], IntervalMixin):
 # int gets hit first and so the correct type is returned
 @overload
 def interval_range(  # pyright: ignore[reportOverlappingOverload]
-    start: int = ...,
-    end: int = ...,
-    periods: int | None = ...,
-    freq: int | None = ...,
-    name: Hashable = ...,
-    closed: IntervalClosedType = ...,
-) -> IntervalIndex[Interval[int]]: ...
-
-# Overlaps since int is a subclass of float
-@overload
-def interval_range(  # pyright: ignore[reportOverlappingOverload]
-    start: int,
-    *,
-    end: None = ...,
-    periods: int | None = ...,
-    freq: int | None = ...,
-    name: Hashable = ...,
-    closed: IntervalClosedType = ...,
-) -> IntervalIndex[Interval[int]]: ...
```

Comment:
having corrected the first overload, this one would never be matched, so it can be removed